lib/android.js

```diff
@@ -1271,8 +1272,13 @@ class ArtStackVisitor {
 
     const visitor = Memory.alloc(baseSize + vtableSize);
 
-    api['art::StackVisitor::StackVisitor'](visitor, thread, context, WalkKind[walkKind], ptr(numFrames),
-      checkSuspended ? 1 : 0);
+    if (pointerSize == 4) {
```

Comment:
```suggestion
    if (pointerSize === 4) {
```

Should always use strict equality checks.